doc/api/net.md

```diff
@@ -2,74 +2,133 @@
 
     Stability: 2 - Stable
 
-The `net` module provides you with an asynchronous network wrapper. It contains
-functions for creating both servers and clients (called streams). You can include
-this module with `require('net');`.
+The `net` module provides an API for implementing network clients and servers.
+It can be accessed using:
+
+```js
+const net = require('net');
+```
 
 ## Class: net.Server
 <!-- YAML
 added: v0.1.90
 -->
 
-This class is used to create a TCP or local server.
+The `net.Server` class is used to implement TCP or local servers.
 
-`net.Server` is an [`EventEmitter`][] with the following events:
+All `net.Server` objects are [`EventEmitter`][] instances.
 
 ### Event: 'close'
 <!-- YAML
 added: v0.5.0
 -->
 
-Emitted when the server closes. Note that if connections exist, this
-event is not emitted until all connections are ended.
+The `'close'` event is emitted when the server closes. If active connections
+exist, this event is not emitted until all such connections are ended.
+
+The listener callback is called without passing any arguments.
+
+```js
+server.on('close', () => {
+  console.log('The server has been closed.');
+});
+```
 
 ### Event: 'connection'
 <!-- YAML
 added: v0.1.90
 -->
 
-* {net.Socket} The connection object
+The `'connection'` event is emitted when a new connection is established.
 
-Emitted when a new connection is made. `socket` is an instance of
-`net.Socket`.
+The listener callback is called with the `net.Socket` object for the connection
+passed as the only argument.
+
+```js
+server.on('connection', (socket) => {
+  console.log('A connection has been established');
+});
+```
 
 ### Event: 'error'
 <!-- YAML
 added: v0.1.90
 -->
 
-* {Error}
+The `'error'` event is emitted when an error occurs. The [`'close'`][] event
+will be emitted immediately following the `'error'` event.
+
+The listener callback is called with the `Error` object passed as the only
+argument.
 
-Emitted when an error occurs.  The [`'close'`][] event will be called directly
-following this event.  See example in discussion of `server.listen`.
+```js
+server.on('error', (error) => {
+  console.log('An error occurred!', error);
+});
+```
 
 ### Event: 'listening'
 <!-- YAML
 added: v0.1.90
 -->
 
-Emitted when the server has been bound after calling `server.listen`.
+The `'listening'` event is emitted when the server has been bound by calling the
+`server.listen()` method.
+
+The listener callback is called without passing any arguments:
```

Comment:
How about

> The callback is passed no arguments.

I think "listener callback" is confusing, because its meant that the callback is added as an event listener, which is true, but it looks like it was intended to say "The `'listening'` callback ...".